build_library/vm_image_util.sh

```diff
@@ -945,9 +964,17 @@ EOF
 }
 
 _write_ova_bundle() {
-    vmdk-convert ${VM_DST_IMG} ${VM_TMP_DIR}/vm.vmdk
-    ( cd $(_dst_dir) && mkova.sh $(_dst_name) ${VM_TMP_DIR}/vm.vmdk ${BUILD_LIBRARY_DIR}/template.ovf)
-    VM_GENERATED_FILES+=$(_dst_dir)/$(_dst_name ".ova")
+    local name=$(_dst_name)
+    local src_name=$(basename "$VM_SRC_IMG")
+    local ovf="$(_dst_dir)/$(_src_to_dst_name "${src_name}" ".ovf")"
+    local mf="${VM_TMP_DIR}/${name}.mf"
+
+    echo "SHA1(${name}-disk1.vmdk)= $(sha1sum ${VM_DST_IMG} | cut -d' ' -f1)" > "${mf}"
+    echo "SHA1(${name}.ovf)= $(sha1sum ${VM_DST_IMG} | cut -d' ' -f1)" >> "${mf}"
+
+    tar -cf $(_dst_path ".ova") "${ovf}" "${mf}" "${VM_DST_IMG}"
+
+    VM_GENERATED_FILES+=( $(_dst_path ".ova") )
```

Comment:
Since you want the VMDK now now you need:

    VM_GENERATED_FILES+=( "$(_dst_path ".ova")" "${VM_DST_IMG}" )

when bundle_format is set DST_IMG is not added to the file list automatically. (see line 345)